Python unit test failure in test_style_transfer

Reproduced on Linux on Python 2.7. May reproduce elsewhere.

```
_____________________ StyleTransferTest.test_export_coreml _____________________

self = <turicreate.test.test_style_transfer.StyleTransferTest testMethod=test_export_coreml>

    def test_export_coreml(self):
        import coremltools
        filename = tempfile.mkstemp('my_style_transfer.mlmodel')[1]
        model = self.model
        model.export_coreml(filename)
    
        coreml_model = coremltools.models.MLModel(filename)
        img = self.style_sf[0:2][self.style_feature][0]
        img_fixed = tc.image_analysis.resize(img, 256, 256, 3)
        img = self._coreml_python_predict(coreml_model, img_fixed)
>       self.assertEqual(img.shape, (256, 256, 3))
E       AttributeError: 'NoneType' object has no attribute 'shape'

test_style_transfer.py:220: AttributeError
```

--- a/src/unity/python/turicreate/test/test_style_transfer.py
+++ b/src/unity/python/turicreate/test/test_style_transfer.py
@@ -214,16 +214,18 @@ class StyleTransferTest(unittest.TestCase):
         model.export_coreml(filename)
 
         coreml_model = coremltools.models.MLModel(filename)
-        img = self.style_sf[0:2][self.style_feature][0]
-        img_fixed = tc.image_analysis.resize(img, 256, 256, 3)
-        img = self._coreml_python_predict(coreml_model, img_fixed)
-        self.assertEqual(img.shape, (256, 256, 3))
 
-        # Test for flexible shape
-        img = self.style_sf[0:2][self.style_feature][1]
-        img_fixed = tc.image_analysis.resize(img, 512, 512, 3)
-        img = self._coreml_python_predict(coreml_model, img_fixed)
-        self.assertEqual(img.shape, (512, 512, 3))
+        if _mac_ver() >= (10, 13):
+            img = self.style_sf[0:2][self.style_feature][0]
+            img_fixed = tc.image_analysis.resize(img, 256, 256, 3)
+            img = self._coreml_python_predict(coreml_model, img_fixed)
+            self.assertEqual(img.shape, (256, 256, 3))
+
+            # Test for flexible shape
+            img = self.style_sf[0:2][self.style_feature][1]
+            img_fixed = tc.image_analysis.resize(img, 512, 512, 3)
+            img = self._coreml_python_predict(coreml_model, img_fixed)
+            self.assertEqual(img.shape, (512, 512, 3))
 
         # Also check if we can train a second model and export it (there could
         # be naming issues in mxnet)

